singer-commons/example/loggingaudit_client_config_thriftlogger_example.properties

```diff
@@ -0,0 +1,42 @@
+###############################################################################
+#
+# config for loggingaudit client
+#
+###############################################################################
+
+
+stage=thriftlogger
+enableAuditForAllTopicsByDefault=false
+queueSize=100000
+enqueueWaitInMilliseconds=0
+
+sender.type=kafka
+sender.kafka.topic=logging_audit
+sender.kafka.producerConfig.bootstrap.servers=testkafka-001:9092,testkafka-002:9092,testkafka-003:9092
+sender.kafka.producerConfig.acks=-1
+sender.kafka.producerConfig.retries=5
+sender.kafka.producerConfig.producer.type=async
+sender.kafka.producerConfig.compression.type=gzip
+sender.kafka.producerConfig.max.request.size=2097152
+sender.kafka.producerConfig.ssl.enabled=true
+sender.kafka.producerConfig.ssl.client.auth=required
+sender.kafka.producerConfig.ssl.enabled.protocols=TLSv1.2,TLSv1.1,TLSv1
+sender.kafka.producerConfig.ssl.endpoint.identification.algorithm=HTTPS
+sender.kafka.producerConfig.ssl.key.password=your_password
+sender.kafka.producerConfig.ssl.keystore.location=/path/to/keystore/location
+sender.kafka.producerConfig.ssl.keystore.password=your_password
+sender.kafka.producerConfig.ssl.keystore.type=JKS
+sender.kafka.producerConfig.ssl.secure.random.implementation=SHA1PRNG
+sender.kafka.producerConfig.ssl.truststore.location=/path/to/truststore/location
+sender.kafka.producerConfig.ssl.truststore.password=your_password
+sender.kafka.producerConfig.ssl.truststore.type=JKS
+
+auditedTopics.names=test1,test2,test3
+auditedTopics.test1.samplingRate=0.01
+auditedTopics.test1.startAtCurrentStage=true
```

Comment:
Both options, stopAtCurrentStage (startAtCurrentStage) and stop_stage (start_stage),  were considered initially.  Choosing stopAtCurrentStage over stop_stage is because there can be different systems (in addition to ThriftLogger, Singer Merced) importing this library for auditing and the number of use cases can increase,  thus we don't know what will be added or removed in the future.  For example, certain streaming app (kafka streams, flink, spark stream)  interacting with Kafka can join the auditing as well.  Thus, I feel specifying stopAtCurrentStage rather stop_stage can provide more flexibility and extensibility.

Reply:
Thanks for the explanation! Could you include this  in the comment?  that will help later readers to understand the design thoughts.